client/src/client/models.py

```diff
@@ -16,6 +18,15 @@ def make_table_name(model_class: Type[peewee.Model]) -> str:
     return model_name.lower() + "s"
 
 
+def default_message_id() -> str:
+    """Create a default id for a message."""
+    return hashlib.sha224(
```

Comment:
What is the reason to use sha224? Later on actual cookies are going be sha256

Reply:
I changed it while testing default values in id because I thought it might be the cause. It wasn't but I forgot to change it later. I'll replace it with sha-256.